components/script.tsx

```diff
@@ -120,6 +120,8 @@ const Script: React.FC<ScriptProps> = ({ file, className, messagesHeight = 'h-fu
 							backButton={hasParams}
 							noChat={!tool.chat}
 							onRestart={restartScript}
+                            onInterrupt={interrupt}
```

Comment:
whys the indentation different on these lines?

Reply:
Its just github displaying it weird. If you view the file its normal